vulnerabilities/scraper/npm.py

```diff
@@ -47,30 +47,27 @@ def remove_spaces(x):
     return x
 
 
-def get_all_version(package_name):
+def get_all_versions(package_name):
     """
-    Returns all available for a module
+    Returns all versions available for a module
     """
-    package_url = NPM_URL.format('/'+package_name)
-    response = urlopen(package_url).read()
-    data = json.loads(response)
-    versions = data.get('versions', {})
-    all_version = [obj for obj in versions]
-    return all_version
+    package_url = NPM_URL.format(f'/{package_name}')
+    data = json.load(urlopen(package_url))
+    return [v for v in data.get('versions', {})]
 
 
-def extract_version(package_name, aff_version_range, fixed_version_range):
+def extract_versions(package_name, aff_version_range, fixed_version_range):
     """
-    Seperate list of Affected version and fixed version from all version
-    using the range specified
+    Seperate list of affected versions and fixed versions from all version
```

Comment:
nitpicking: `Separate list of affected versions and fixed versions from all versions`

Reply:
Also unrelated, could it make sense to adopt "affect/fixed" rather "impacted/resolved" ? is this a better wording and more explicit?

Reply:
I think either is fine. We should make sure to be consistent in the data we store, API responses, documentation, etc. but I'm fine sticking with impacted/resolved. If you are in favor of affected/fixed we should change it now though, since this PR already resets migrations.